### ISSUE TITLE
WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (.sp-c-fuh-social--youtube)

### ISSUE THREAD

**Issue by simonellis660** [simonellis660/Test#1]
#### Issue
`Ensures links have discernible text` ([link-name](https://accessibilityinsights.io/info-examples/web/link-name))
#### Target application
[Home - BBC Sport](https://www.bbc.co.uk/sport)
#### Element path
.sp-c-fuh-social--youtube
#### Snippet
`<a href="https://www.youtube.com/channel/UCW6-BQWFA70Dyyc7ZpZ9Xlg" class="sp-c-fuh-social sp-c-fuh-social--youtube" data-reactid=".2595i44jmv8.1.1.0.0.0.1.0.3">`
#### How to fix
    Fix all of the following:
      Element is in tab order and does not have accessible text
    
    Fix any of the following:
      Element does not have text that is visible to screen readers
      aria-label attribute does not exist or is empty
      aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
      Element has no title attribute
#### Environment
Chrome version 92.0.4515.159
This accessibility issue was found using Accessibility Insights for Web 2.27.0 (axe-core 4.2.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
